vendor/bat-native-ledger/src/bat/ledger/internal/media/vimeo_unittest.cc

```diff
@@ -14,10 +14,202 @@ namespace braveledger_media {
 class MediaVimeoTest : public testing::Test {
 };
 
+const char profile_html[] =
```

Comment:
I'm not too picky so this is up to you, but this seems big enough that you might want to put it in a data file that the test reads. It would make it easier to update in the future too, if that's a concern.